scalableminds/webknossos
# configurable organization logos

- logo which is displayed next to tracing view is now configurable
- datasets and organizations now have a logoUrl property
- organization logoUrl is used as fallback if datasets logoUrl is `null`

### URL of deployed dev instance (used for testing):
- https://organizationlogos.webknossos.xyz/

### Steps to test:
- per default the mpi logo should be displayed
- change the dataset logoUrl in DB to another logo
- new logo should be shown for that dataset
- change back the logoUrl to null
- mpi logo should be shown again
- change the org logo
- the new logo should be used as fallback

### Issues:
- fixes #2522 

------
